sale_order_line_description/tests/test_sale.py

```diff
@@ -52,20 +52,12 @@ def _create_user(self, name, group=None):
 
     def test_check_sale_order_line_description(self):
         line_values = {"order_id": self.sale_order.id, "product_id": self.product.id}
-
         # Create sale order line with TestUser1
-        sale_order_line = self.sale_order_line_model.sudo(self.user_1).create(
+        sale_order_line = self.sale_order_line_model.with_user(self.user_1).create(
             line_values.copy()
         )
-        self.assertEqual(
```

Comment:
Why removing a test that seems to reflect module functionnality ?

Reply:
@rousseldenis at first I don't understand why we need to check it, but I think carefully it's make sense to check it, so I'll fix it.

thank for your review.